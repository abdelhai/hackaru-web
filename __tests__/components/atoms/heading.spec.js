import Factory from '@/__tests__/__setups__/factory';
import Heading from '@/components/atoms/heading';

describe('Heading', () => {
  it('render correctly', () => {
    const wrapper = new Factory(Heading, {
      slots: {
        default: '<h1>Title</h1>'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
