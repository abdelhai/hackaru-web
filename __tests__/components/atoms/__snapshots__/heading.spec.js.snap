// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading render correctly 1`] = `
<h1
  class="heading"
>
  <h1>
    Title
  </h1>
</h1>
`;
